storage/src/test/java/tech/pegasys/teku/storage/client/RecentChainDataTest.java

```diff
@@ -63,10 +63,11 @@
 import tech.pegasys.teku.storage.store.UpdatableStore;
 import tech.pegasys.teku.storage.store.UpdatableStore.StoreTransaction;
 import tech.pegasys.teku.util.EventSink;
-import tech.pegasys.teku.util.config.Constants;
 
 class RecentChainDataTest {
-  private final DataStructureUtil dataStructureUtil = new DataStructureUtil();
+  private final SpecProvider specProvider = StubSpecProvider.createMinimal();
+  private final DataStructureUtil dataStructureUtil = new DataStructureUtil(specProvider);
+  private final SpecConstants specConstants = specProvider.getGenesisSpecConstants();
```

Comment:
Probably a good idea to explicitly name this `genesisConstants` since constants can change over time
```suggestion
  private final SpecConstants genesisConstants = specProvider.getGenesisSpecConstants();
```